python/pydantic_core/core_schema.py

```diff
@@ -2458,6 +2478,7 @@ class UnionSchema(TypedDict, total=False):
     type: Required[Literal['union']]
     choices: Required[List[Union[CoreSchema, Tuple[CoreSchema, str]]]]
     # default true, whether to automatically collapse unions with one element to the inner validator
+    tagged_union_tag: str
```

Comment:
Was this something missing before?

Reply:
No, should be removed, thanks for catching this